release/platform/platform.go

```diff
@@ -0,0 +1,135 @@
+package platform
+
+import (
+	"errors"
+	"fmt"
+	"os"
+)
+
+// Platform represents a platform (a combination of OS, distro,
+// version, and architecture) on which we may build/test the tools.
+// There should be at least one evergreen buildvariant per platform,
+// and there may be multiple.
+type Platform struct {
+	Name string
+	Arch string
+}
+
+const evgPlatformVar = "EVG_PLATFORM"
+
+// Get returns the Platform for this host, based on the value of
+// EVG_PLATFORM. It returns an error if EVG_PLATFORM is unset or set
+// to an unknown value.
+func Get() (Platform, error) {
+	evgPlatform := os.Getenv(evgPlatformVar)
+	if evgPlatform == "" {
+		return Platform{}, errors.New("EVG_PLATFORM not set")
```

Comment:
minor: if you're going to have a constant on line 18, you should probably use it here, too, for consistency in case it ever changes.

Reply:
done!